src/components/Alerts/ExpenseAddedAlert.js

```diff
@@ -1,16 +1,16 @@
-const ExpenseAddedAlert = ({ expenseAdded, setExpenseAdded }) => {
+const ExpenseAddedAlert = ({ expenseWasAdded, setShowExpenseAddedAlert }) => {
```

Comment:
these alerts seem very repetitive. can we simplify/generalize similar to modals?

Reply:
Sure, I can take Giulio's lead and abstract the alerts to a more general `<Alert />` component like the modals.

Reply:
Done. Please let me know if you think this works well.

Reply:
much better! now we only need to find a solution for the state variables